recipes/libpgmath/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "libpgmath" %}
+{% set version = "2019.08.01" %}
```

Comment:
Is this going to cause problems with future version numbers?

Reply:
No, they have version numbers like that (last release version number is 2019.03.29), but my PR for windows and osx support landed recently and I want to have this package now to play with it.

Reply:
Ok, sounds good